### PR TITLE
 fix(frontend): SQL server 集群管理 #2996

### PR DESCRIPTION
**Pull request by styLjc**

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-dbm?pullRequest=3213) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-dbm?pullRequest=3213) before we can accept your contribution.<br/><hr/>**v_ljcgliu(刘江城)** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/blueking-dbm?pullRequest=3213) it.</sub>

File: dbm-ui/frontend/src/services/clusters.ts [hunk 1]
```diff
@@ -117,3 +117,10 @@ export const updateClusterEntryConfig = (params: {
     target_instances: string[],
   }[]
 }) => http.post<{ cluster_id?: number }>('/apis/cluster_entry/refresh_cluster_domain/', params);
+
```
Comment (3octaves) [inline dbm-ui/frontend/src/services/clusters.ts:120]: 新增接口需要放在service/source的相应模块

File: dbm-ui/frontend/src/views/sqlserver-manage/apply/SqlServer.vue [hunk 2]
```diff
@@ -0,0 +1,719 @@
...
+            <DbIcon
+              v-bk-tooltips="t('刷新获取最新DB模块名')"
+              class="spec-refresh-icon ml-6"
+              type="refresh"
+              @click="getModulesConfig" />
+            <div class="apply-form-database">
+              <BkLoading :loading="isModuleLoading">
+                <div v-if="levelConfigData && levelConfigData.conf_items.length">
+                  <div
+                    v-for="(item, index) in levelConfigData?.conf_items"
```
Comment (3octaves) [inline dbm-ui/frontend/src/views/sqlserver-manage/apply/SqlServer.vue]: 前面if有判断，此处的？似乎可省略

File: dbm-ui/frontend/src/views/sqlserver-manage/apply/components/DomainTable.vue [hunk 3]
```diff
@@ -126,19 +126,20 @@
...
-        <span>
+         <span>
           {
             props.isSqlserverSingle ? t('域名') : t('主域名')
           }
           {
-            tableData.value.length !== 0
-            && <BatchEdit
+            tableData.value.length !== 0 && (
+              <BatchEdit
```
Comment (jinquantianxia) [inline dbm-ui/frontend/src/views/sqlserver-manage/apply/components/DomainTable.vue:135]: 空格与对齐

File: dbm-ui/frontend/src/views/sqlserver-manage/single-cluster/components/List.vue [hunk 4]
```diff
@@ -187,32 +210,38 @@
...
+                v-bk-tooltips={ t('复制主访问入口') }
+                onClick={ () => copy(data.master_domain) } />
+              <db-icon
+                type="link"
+                v-bk-tooltips={ t('新开tab打开') } />
+              <div class="text-overflow" v-overflow-tips>
+                {
+                  data.isNew && (
+                    <span
+                    class="glob-new-tag cluster-tag ml-4"
```
Comment (jinquantianxia) [inline dbm-ui/frontend/src/views/sqlserver-manage/single-cluster/components/List.vue]: 空格

**Pull request closed without merging**